modules/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+plugins {
+    `java-platform`
+}
+
+javaPlatform {
+    allowDependencies()
+}
+
+dependencies {
+    // This platform depends on each of its subprojects.
+    subprojects {
+        runtime(this)
+    }
+}
+
+// Allows using :modules:clean as a shortcut for running clean in each module.
+tasks.named("clean").configure {
+    val cleanPlatform = this
```

Comment:
Again understanding what this does, but `cleanPlatform` throws me off naming-wise. Is it almost like "cleanPlatform" would really be the task _name_? But since you want `:modules:clean` to work then the task name just remains "clean"

Would something like `cleanAllModules` and `cleanAllModules.dependsOn(...` do the same thing but be clearer in naming?

Reply:
`cleanPlatform` here is a local variable so I can use it below after `this` has been shadowed by the subproject. It's not renaming any tasks.

Reply:
Oh yes, I got that part. Thus the "almost like" 😁 

Like in the abstract this class could as well have been called "cleanPlatform" or "cleanAllModules" - plain "cleanPlatform" just throws me off as a variable name the way it gets used, but that's very minor and probably just me 👍